deptofdefense/move.mil
# Linehaul single table inheritance

## Checklist

I have…


## Summary of Changes

Refactors the Linehaul data models so that the CONUS and Intra-AK linehaul models inherit from a common BaseLinehaul model via Single Table Inheritance (STI).

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `linehaul-type-inheritance`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in the Web browser of your choice.
